source/index.d.ts

```diff
@@ -0,0 +1,23 @@
+import {ClientRequest} from 'http';
+
+export interface Timings {
+	start: number;
+	socket?: number;
+	lookup?: number;
+	connect?: number;
+	upload?: number;
+	response?: number;
+	end?: number;
+	error?: number;
+	phases: {
+		wait?: number;
+		dns?: number;
+		tcp?: number;
+		request?: number;
+		firstByte?: number;
+		download?: number;
+		total?: number;
+	}
+}
+
+export default function(request: ClientRequest): Timings;
```

Comment:
You need to follow the style I mentioned: https://github.com/sindresorhus/log-update/blob/master/index.d.ts Meaning no `export default`. This is to preserve CommonJS compatibility.